src/cli/onefuzz/backend.py

```diff
@@ -322,6 +345,8 @@ def request(
             raise Exception("endpoint not configured")
 
         url = endpoint + "/api/" + path
+
+        self.config_params()
```

Comment:
Why do we update the config on each request ? 
It does not have to be done in this PR but if the config is already set there is no need fetch the config again from the server

Reply:
The goal is to not store the config values locally.

Reply:
Let's have `config_params` check  if it has the local values first before issuing the request.

Reply:
I think this is a good idea. I'll get that going.

Reply:
> Let's have `config_params` check if it has the local values first before issuing the request, if it has all of them there's no need to go fetch them from the server. This makes it okay to invoke `config_params` on each request; the first time it will fetch the values if needed and then they will be used for the rest of the lifetime of the current CLI command.

i think this is good first step but ultimatelyy i think we still need to store the config in a file to avoid having to fetch the same config for every commands. This will be use full for apps that uses the cli for scripting.